public/app/features/alerting/unified/components/receivers/form/GrafanaReceiverForm.tsx

```diff
@@ -129,8 +130,8 @@ export const GrafanaReceiverForm = ({ existing, alertManagerSourceName, config }
   // or a Mimir-based AlertManager
   const isManageableAlertManagerDataSource = !isVanillaPrometheusAlertManagerDataSource(alertManagerSourceName);
```

Comment:
`isManageableAlertmanagerDataSource` in `CloudReceiverForm` combines `readonly` and `isVanillaProm...`
Should we unify the meaning of `isManageableAlertmanagerDataSource` between the two?

Reply:
Yea I think there is some duplication here, not sure why I introduced the new prop here instead of just using the abilities (alertmanager with config API and RBAC) 🤔